x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authc/ldap/LdapTestUtils.java

```diff
@@ -19,56 +18,47 @@
 import org.elasticsearch.xpack.core.ssl.VerificationMode;
 import org.elasticsearch.xpack.security.authc.ldap.support.LdapUtils;
 
-import java.nio.file.Path;
+import java.util.List;
 
 public class LdapTestUtils {
 
     private LdapTestUtils() {
         // Utility class
     }
 
-    public static LDAPConnection openConnection(String url, String bindDN, String bindPassword, Path truststore) throws Exception {
-        boolean useGlobalSSL = ESTestCase.randomBoolean();
+    public static LDAPConnection openConnection(String url, String bindDN, String bindPassword, List<String> certificateAuthorities) throws
+        Exception {
```

Comment:
for the line wrap can you split it right before `List<String`? I find this easier to read